### DISCUSSION TITLE
[Report]官网无法评论

### DISCUSSION THREAD

**Discussion by dyxang** [walinejs/waline#78] [Q&A] [answered]
直接访问 https://waline.vercel.app/ 显示

Application Error
The error occurred in the hosted application
~~Platform Error~~
~~The error occurred in the infrastructure layer~~
**402**: PAYMENT_REQUIRED
Code: DEPLOYMENT_DISABLED

**Comment by lizheming** [comment, marked as answer]
嗯，已知情况，我的 Vercel 个人账号超额了，在等恢复中。
